devtools.gemspec

```diff
@@ -17,6 +17,11 @@ Gem::Specification.new do |gem|
   gem.extra_rdoc_files      = %w[README.md TODO]
   gem.required_ruby_version = '>= 2.1'
 
+  gem.add_runtime_dependency 'procto', '~> 0.0.x'
+  gem.add_runtime_dependency 'anima', '~> 0.3.x'
+  gem.add_runtime_dependency 'concord', '~> 0.1.x'
+  gem.add_runtime_dependency 'adamantium', '~> 0.2.x'
+
   gem.add_dependency 'rspec',        '~> 3.3.0'
   gem.add_dependency 'rspec-core',   '~> 3.3.0'
   gem.add_dependency 'rspec-its',    '~> 1.2.0'
```

Comment:
Lets name these`add_runtime_dependency` instead of `add_dependency` for consistency.